src/main/webapp/js/adminEmail.es6

```diff
@@ -1,81 +1,52 @@
 
 // Form input placeholders
-var PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image';
+const PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image';
 
-var callbackFunction;
+let callbackFunction;
 
-$(document).ready(function() {
-
-    $('.navbar-fixed-top').css('zIndex', 0);
-
-    /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
-    richTextEditorBuilder.initEditor('textarea', {
-        document_base_url: $('#documentBaseUrl').text(),
-        file_picker_callback: function(callback, value, meta) {
-
-            // Provide image and alt text for the image dialog
-            if (meta.filetype === 'image') {
-                $('#adminEmailFile').click();
-                callbackFunction = callback;
-            }
-        }
-    });
-    /* eslint-enable camelcase */
-
-    $('#adminEmailFile').on('change paste keyup', function() {
-        createImageUploadUrl();
-    });
-
-    $('#adminEmailGroupReceiverList').on('change paste keyup', function() {
-        createGroupReceiverListUploadUrl();
-    });
-
-    $('#adminEmailGroupReceiverListUploadButton').on('click', function() {
-        $('#adminEmailGroupReceiverList').click();
-    });
-
-    $('#composeSaveButton').on('click', function() {
-        $('#adminEmailMainForm').attr('action', '/admin/adminEmailComposeSave');
-        $('#composeSubmitButton').click();
-    });
-
-    $('#addressReceiverEmails').on('change keyup', function(e) {
-        if (e.which === 13) {
-            $('#addressReceiverEmails').val($('#addressReceiverEmails').val() + ',');
-        }
-    });
+function setErrorMessage(message) {
+    setStatusMessage(message, StatusType.DANGER);
+}
 
-    toggleSort($('#button_sort_date').parent());
-});
+function showUploadingGif() {
+    setStatusMessage("Uploading...<span><img src='/images/ajax-loader.gif'/></span>", StatusType.WARNING);
+}
 
 function createGroupReceiverListUploadUrl() {
-
     $.ajax({
         type: 'POST',
         url: '/admin/adminEmailCreateGroupReceiverListUploadUrl',
-        beforeSend: function() {
+        beforeSend() {
             showUploadingGif();
         },
-        error: function() {
+        error() {
             setErrorMessage('URL request failured, please try again.');
         },
-        success: function(data) {
-            setTimeout(function() {
+        success(data) {
+            setTimeout(() => {
                 if (data.isError) {
                     setErrorMessage(data.ajaxStatus);
                 } else {
                     $('#adminEmailReceiverListForm').attr('action', data.nextUploadUrl);
                     setStatusMessage(data.ajaxStatus);
-                    submitGroupReceiverListUploadFormAjax();
+                    submitGroupReceiverListUploadFormAjax();    // eslint-disable-line no-use-before-define
                 }
             }, 500);
```

Comment:
Not sure if its a good idea to remove the 500 ms delay here and in other places? It probably won't break anything, but maybe might be better to minimize functionality changes.